phpcs.xml.dist

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0"?>
+<ruleset name="Beans">
+    <description>The code standard for Beans.</description>
+    <file>.</file>
+    <exclude-pattern>dist/*</exclude-pattern>
+    <exclude-pattern>node_modules/*</exclude-pattern>
+    <exclude-pattern>vendor/*</exclude-pattern>
+    <arg name="extensions" value="php"/>
+    <arg value="sp"/>
+    <arg name="report" value="full"/>
+    <arg name="report" value="summary"/>
+    <arg name="report" value="source"/>
+    <rule ref="PHPCompatibility" />
+    <config name="testVersion" value="5.2-"/>
+    <rule ref="WordPress">
+        <exclude name="WordPress.VIP"/>
+    </rule>
+    <rule ref="WordPress.Files.FileName">
+        <properties>
+            <property name="is_theme" value="true"/>
+            <property name="strict_class_file_names" value="false"/>
+        </properties>
+        <exclude name="WordPress.Files.FileName.NotHyphenatedLowercase"/>
+    </rule>
+    <rule ref="WordPress.NamingConventions.PrefixAllGlobals">
+        <properties>
+            <property name="prefixes" type="array" value="beans,_beans,_Beans" />
+        </properties>
+    </rule>
+    <rule ref="WordPress.WhiteSpace.ControlStructureSpacing">
+        <properties>
+            <property name="blank_line_check" value="true" />
+        </properties>
+    </rule>
+    <rule ref="WordPress.WP.DeprecatedFunctions">
+        <properties>
+            <property name="minimum_supported_version" value="4.8" />
+        </properties>
+    </rule>
+    <rule ref="WordPress.WP.I18n">
+        <properties>
+            <property name="text_domain" type="array" value="tm-beans" />
```

Comment:
This is a problem. The text domain should match the theme slug (i.e. the directory it would be installed as under `wp-content/themes`. Presumably, this would be `beans`, so all of the text domain instances should be updated, and this value changed.

Reply:
@GaryJones You are correct.  In versions <2.0, beans _should be_ installed as `tm-beans` and not `beans` within the `wp-content/themes` folder. If not, any child themes that extend as `tm-beans` will get an error:

>### Broken Themes
>The following themes are installed but incomplete.

|**Name**   |    **Description**   |
|------------| -------------------|
|Beans child |  The parent theme is missing. Please install the "tm-beans" parent theme. |

## Mark as new issue

I opened [issue #59](https://github.com/Getbeans/Beans/issues/59) to propose changing the text domain and theme folder name to `beans` in version 2.0.

## Version < 2.0

In versions <2.0, we'll stick with `tm-beans`.  And yes, the theme's folder name is also `tm-beans`.